packages/web-components/src/global/variables.css

```diff
@@ -279,6 +279,7 @@
   --ic-space-xxl: 3rem;
 
   /* z-index */
+  --ic-z-index-sidebar: 1200;
```

Comment:
Might we want to call this '--ic-z-index-side-nav' to match the component name? Although if the purpose of creating this variable was for #391 then I wonder if something like '--ic-z-index-nav-menu' might be better as it's more generic, and we could use that for the top navigation menu as well for example when we come to that ticket

Reply:
Agreed, will rename to nav-menu